com.unity.render-pipelines.high-definition/Runtime/RenderPipelineResources/HDProjectSettingsProxy.cs

```diff
@@ -0,0 +1,13 @@
+#if UNITY_EDITOR
+using System;
+#endif
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+#if UNITY_EDITOR
+    static class HDProjectSettingsProxy
+    {
+        public static Func<string> projectSettingsFolderPath;
+    }
```

Comment:
@sebastienlagarde See HDProjectSettings.cs returned to editor assembly and the InitializeOnLoadMethod method.
@RSlysz for visibility.